src/WellPicker.tsx

```diff
@@ -112,7 +121,9 @@ export const MultiWellPicker: FunctionComponent<IWellPickerProps> = ({
           throw new Error('invalid range selection mode');
         }
       }
-      setBooked(new Set(range.map((label) => wellPlate.getIndex(label))));
+      setBooked(
+        new Set(range.map((label) => wellPlate.getPosition(label, 'index'))),
```

Comment:
The new api allows you to remove this map by asking for a position by index in `getPositionSubset`